### DISCUSSION TITLE
Use Git LFS for Binary Data (Regression Images / Sample Map Data)

### DISCUSSION THREAD

**Discussion by inforithmics** [Mapsui/Mapsui#2750] [Development]
See

https://git-lfs.com/

With Git Lfs Binary Data is only saved as a Pointer to an external Storage making the Git Repository Samller and faster.

**Comment by pauldendulk** [comment]
A downside may be that anyone cloning the git repository would also need to `git lfs install` to run the tests. Still, it looks like the best option for us.
